frontend/simple/views/components/Voting.vue

```diff
@@ -0,0 +1,292 @@
+<template>
+  <div class="is-flex gi-voting">
+    <div class="gi-voting-body is-flex">
+      <div class="gi-voting-sign">
+        <svg class="gi-voting-sign-svg"
+          v-if="isTypeRule"
+        >
+          <circle cx="36" cy="36" r="35"
+            class="gi-voting-sign-svg-circle"
+            :style="svgCircle.style"
+            :class="svgCircle.class"
+          />
+        </svg>
+
+        <p class="gi-voting-sign-value title is-size-4 has-text-centered"
+          :class="{
+            'gi-is-mincome': isTypeMincome,
+            'gi-is-rule': isTypeRule
+          }"
+          v-if="isTypeRuleOrMincome"
+        >
+          {{proposalValue}}
+        </p>
+
+        <img class="gi-voting-sign-avatar"
+          :src="proposal.member.picture"
+          :alt="`${proposal.member.name}'s avatar`"
+          v-if="isTypeMember"
+        />
+      </div>
+
+      <!-- TODO: fix the i18n stuff here -->
+      <div class="gi-voting-info">
+        <h5 class="has-text-weight-bold is-uppercase gi-voting-info-title">{{proposal.title}}</h5>
+        <p v-html="proposal.text"></p>
+        <p class="is-size-7 has-text-grey" v-if="proposal.textDetails" v-html="proposal.textDetails"></p>
+      </div>
+    </div>
+
+    <div class="gi-voting-ctas">
+      <div class="buttons">
+        <template v-if="!proposal.ownProposal">
```

Comment:
a template inside a template? what does this do? :)

Reply:
This is really cool, I discovered it while I was struggling to avoid `<span> /<div>` wrappers just because of conditionals. [Conditional Groups with v-if on template](https://vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt)

Reply:
wow, cool! 👍